### PR TITLE
use element identifier for both paths

### PR DESCRIPTION
**Pull request by hexylena**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

I had a user complain of their taxonomy coming out with `0,1,2,3` as the sample names which they found difficult, and I noticed the two input pathways weren't unified, one used element identifier. So this just pulls out the shared portions and uses element identifier in both paths.

### CODE REVIEW

File: tools/taxonomy_krona_chart/taxonomy_krona_chart.xml [hunk 1]
```diff
@@ -79,7 +74,30 @@
...
       <param name="combine_inputs" value="False"/>
-      <output name="output" file="krona_test1.html" ftype="html" lines_diff="102"/>
+      <output name="output" ftype="html">
+        <assert_contents>
+          <has_text text="KRONA"/>
+          <has_text text="taxonomy_data"/>
+          <has_text text="Eukaryota"/>
+          <has_text text="Chordata"/>
+          <has_text text="Rodentia"/>
+          </assert_contents>
```
Comment (nsoranzo) [inline tools/taxonomy_krona_chart/taxonomy_krona_chart.xml:84]:
```suggestion
        </assert_contents>
```

File: tools/taxonomy_krona_chart/taxonomy_krona_chart.xml [hunk 2]
```diff
@@ -79,7 +74,30 @@
...
+      <param name="max_rank" value="Genus"/>
+      <param name="root_name" value="Root"/>
+      <param name="combine_inputs" value="False"/>
+      <output name="output" ftype="html">
+        <assert_contents>
+          <has_text text="KRONA"/>
+          <has_text text="taxonomy_data"/>
+          <has_text text="tax2"/>
+          <has_text text="Proteobacteria"/>
+          </assert_contents>
```
Comment (nsoranzo) [inline tools/taxonomy_krona_chart/taxonomy_krona_chart.xml:99]:
```suggestion
        </assert_contents>
```

File: tools/taxonomy_krona_chart/taxonomy_krona_chart.xml [hunk 3]
```diff
@@ -79,7 +74,30 @@
       <param name="max_rank" value="Genus"/>
       <param name="root_name" value="Root"/>
       <param name="combine_inputs" value="False"/>
-      <output name="output" file="krona_test1.html" ftype="html" lines_diff="102"/>
```
Comment (nsoranzo) [inline tools/taxonomy_krona_chart/taxonomy_krona_chart.xml:82]: Can you remove `krona_test1.html` from `test-data/` , now it's unused.

**Review by nsoranzo — APPROVED** [review]

**Comment by hexylena** [comment]
thanks!

**Pull request merged**